Module4.3_Search_and_Sorting_Algorithms/activities/Act3_BinaryHeaps/2.md

```diff
@@ -14,7 +14,33 @@ The next method we will implement is `insert`. The easiest, and most efficient,
 
 Notice that when we percolate an item up, we are restoring the heap property between the newly added item and the parent. We are also preserving the heap property for any siblings. Of course, if the newly added item is very small, we may still need to swap it up another level. In fact, we may need to keep swapping until we get to the top of the tree. 
 
-Below depicts the `percUp` method, which percolates a new item as far up in the tree as it needs to go to maintain the heap property. Here is where our wasted element in `heapList` is important. Notice that we can compute the parent of any node by using simple integer division. The parent of the current node can be computed by dividing the index of the current node by 2.
+Let's look at the `percUp` method, which percolates a new item as far up in the tree as it needs to go to maintain the heap property. 
```

Comment:
introduce the word percolate as it is used here. The word just starts being used and it is not really clear that percolating is synonymous with swapping